base/broadcast.jl

```diff
@@ -850,6 +850,11 @@ function __dot__(x::Expr)
            Meta.isexpr(x.args[1], :call) # function or macro definition
         Expr(x.head, x.args[1], dotargs[2])
     else
+        if x.head == :&& || x.head == :||
+            Base.depwarn("""
+                using $(x.head) expressions in @. is deprecated; in the future it will
+                become elementwise. Break the expression into smaller parts instead.""", ())
```

Comment:
This should still pass some name symbol, maybe `:__dot__`.

Reply:
Yeah, I tried that.  The trouble is then the stackframe is identical to every call, leading to only one depwarn.  This is a hack that when you pass `()`, it'll skip the stack frame lookups and just always warn.

Reply:
I suppose I could change the hack to look at the symbol and always warn if it starts with `@`.  That's probably better.

Reply:
Ok then we should use `nothing` instead of `()`.